### PR TITLE
OSX fixes

### PR DESCRIPTION
**Pull request by Senzaki**
Hi,
Currently the engine is not working on MacOS. Installation (via go get) does not work out-of-the-box, even when the dependencies are installed via Homebrew (the package manager that I think is the most widely used in the dev community). Furthermore, some of the examples crash, due to a call to glLineWidth. glLineWidth has been almost-deprecated in OpenGL 3.3 (it still exists, but will only accept line widths in the range ]0, 1], and those are basically always mapped to a value of 1). See the [specification](https://www.khronos.org/registry/OpenGL/specs/gl/glspec33.core.pdf), page 111.
This PR updates the search paths for Vorbis and OpenAL so that they support default `brew install` locations, updates the README file for instructions about how to set it all up on MacOS, and removes any call to glLineWidth.
This effectively removes the SetLineWidth feature from Material, but this feature was not reliable beforehand anyways. I also have updated the g3nd demos accordingly, and will do a PR there as well.
I could also have removed the content of SetLineWidth but kept it, so that it removes the crash but fails silently, thus keeping backward compatibility, but I'm not sure this is the right thing to do since g3nd is still young. What do you think about this?
Best regards!

### CODE REVIEW

**Comment by Senzaki** [comment]
Oh, and the C code now also prints an error message when it panics instead of the plain OpenGL error code. The messages are taken from [here](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glGetError.xhtml).

**Comment by danaugrs** [comment]
Hi! Thanks so much for the contributions!
We would love for g3n to work on Mac out of the box and we really appreciate the search path changes.

However, after discussing with @leonsal, we don't want to remove glLineWidth from the API.
Based on [this comment](https://github.com/g3n/engine/pull/18#issuecomment-323543045) (and the [OpenGL specification](http://docs.gl/gl3/glLineWidth)), setting glLineWidth to 1 should work on OSX.
If that's the case, we would like to have glLineWidth set to 1 by default [here](https://github.com/g3n/engine/blob/master/graphic/axis_helper.go#L42). That way the engine will remain faithful to the OpenGL specification.

As we understand it, the fact that line widths >1 fail on Mac is an OSX OpenGL implementation gap. Per the spec, ["If an unsupported width is requested, the nearest supported width is used"](http://docs.gl/gl3/glLineWidth) but [Mac throws an error instead](https://github.com/g3n/engine/issues/28#issue-287394251). I think the ideal solution would be to query the range of available widths (as suggested [here](https://github.com/g3n/engine/pull/18#issuecomment-321907133)) during startup and modify our [gls.LineWidth](https://github.com/g3n/engine/blob/master/gls/gls.go#L480) implementation to cap the width to the valid range. Let's do that soon but not as part of this pull request. For now (until that's implemented) line widths >1 will break cross-platform compatibility (Mac users will have problems while Windows/Linux users will be fine using larger widths)

So, in summary:
Could you reinstate the glLineWidth API, and set glLineWidth it to 1 [here](https://github.com/g3n/engine/blob/master/graphic/axis_helper.go#L42)?
We will be happy to merge then!

Thanks again.

**Comment by danaugrs** [comment]
Spoke with @Senzaki and implemented the changes I talked about above in [this commit](https://github.com/g3n/engine/commit/1929d6070113634d2d0ac653b6c3bcf67bf4033a).
